tensorflow/core/kernels/depthtospace_op.cc

```diff
@@ -172,6 +172,16 @@ struct DepthToSpaceOpFunctor<CPUDevice, T, FORMAT_NHWC> {
     }
   }
 };
+
+#ifdef WIN32
+template <typename T>
+struct DepthToSpaceOpFunctor<CPUDevice, T, FORMAT_NCHW> {
+  void operator()(const CPUDevice& d, typename TTypes<T, 4>::ConstTensor input,
+                  int block_size, typename TTypes<T, 4>::Tensor output) {
+    LOG(FATAL) << "dummy implementation to make debug build compile";
```

Comment:
Let's try a wording that is less loaded

```suggestion
    LOG(FATAL) << "Trivial implementation to make debug build compile";
```

Reply:
I rephrased the wording as you suggested, here and at the second (symmetric) location it occurred.

Reply:
Just FYI there's an alternative solution to this using C++ 17 `if constexpr` as mentioned in #42094